cmd/ledger/main.go

```diff
@@ -41,6 +41,8 @@ var (
 	hub          = flag.String("hub", "", "hub address, default to docker hub")
 	imageVersion = flag.String("image-version", "latest", "image version")
 	storageClass = flag.String("storage-class", "local-storage", "storage class name")
+	runTime      = flag.Duration("run-time", 100*time.Minute, "client test run time")
+	round        = flag.Int("round", 1, "client test request round")
```

Comment:
round wasn't used to control the run round in ModeSelfScheduled actually, so I suggest remove this flag and set RunRound to 1 on `cfg` below